Dockerfile

```diff
@@ -50,12 +50,11 @@ RUN apk --no-cache add --virtual \
 
 WORKDIR /app
 
-COPY --from=build /app/node_modules /app/node_modules
-COPY --from=build /app/.next /app/.next
 COPY --from=build /app/next.config.js /app/next.config.js
 COPY --from=build /app/public /app/public
 COPY --from=build /app/package.json /app/package.json
-COPY --from=build /app/yarn.lock /app/yarn.lock
+COPY --from=build --chown=nextjs:nodejs /app/.next/standalone ./
+COPY --from=build --chown=nextjs:nodejs /app/.next/static ./.next/static
 
 EXPOSE 3000
-CMD [ "yarn", "start" ]
+CMD [ "node", "server.js" ]
```

Comment:
Are we moving away from yarn? Why not start the explorer using the [pre-defined package.json script](https://github.com/hirosystems/explorer/blob/main/package.json#L14)?

Reply:
the `standalone` feature produces a minimal file `server.js` which is used to run and it doesn't look like `yarn` can execute this.  Additionally, per their documentation, the `public` and `static` assets should be served by a CDN instead of baking it into the image.  This is fine for now but something definitely worth looking into in the future.

Reply:
Interesting, thanks for clarifying!